package.json

```diff
@@ -79,7 +79,7 @@
   "lint-staged": {
     "*.{ts,js,css}": [
       "prettier --write",
-      "eslint --fix"
+      "eslint --fix --max-warnings 0"
```

Comment:
Changing this so that settings for pre-commit hook match settings run during build (I got caught out by it passing pre-commit hooks but then failing on CircleCI...)